tests/main/failover/task.yaml

```diff
@@ -21,10 +21,14 @@ environment:
     TARGET_SNAP/rclocalcrash: core
     TARGET_SNAP/emptysystemd: core
     #TARGET_SNAP/emptyinitrd: kernel
+    BUILD_DIR: /home/tmp
```

Comment:
We could also use /var/tmp which is also a persistent (non tmpfs) tmp. But this is fine.